frontend/src/views/Apps/AppDetail/AppDetail.js

```diff
@@ -0,0 +1,205 @@
+
+    return (
+      <div id="app-detail">
+        <Row>
+          <Col>
+            <div className="head">
+              {/* eslint-disable-next-line jsx-a11y/alt-text */}
+              <img src={icon} />
+              <div className="info">
+                <h1 className="d-flex align-items-baseline">
+                  {name}
+                  <Badge variant="dark" className="ml-2 pt-2 pb-2 pl-3 pr-3">
+                    Free Tier
+                  </Badge>
+                </h1>
+                <p className="description">{description}</p>
+              </div>
+            </div>
+          </Col>
+        </Row>
+        <h2 className="mt-4">General Information</h2>
+        <Row className="mt-2 stats">
+          {generalInfo.map((card, idx) => (
+            <Col key={idx}>
+              <InfoCard title={card.title} subtitle={card.subtitle} />
+            </Col>
+          ))}
+          <Col>
+            <InfoCard title={jailed === 1 ? "YES" : "NO"} subtitle={"Jailed"}>
+              {/*eslint-disable-next-line jsx-a11y/anchor-is-valid*/}
+              <a className="link" href="#">
+                Take out of jail
+              </a>
+            </InfoCard>
+          </Col>
+        </Row>
+        <Row className="contact-info stats">
+          {contactInfo.map((card, idx) => (
+            <Col key={idx}>
+              <InfoCard
+                className="pl-4"
+                title={card.title}
+                subtitle={card.subtitle}
+              >
+                <span></span>
+              </InfoCard>
+            </Col>
+          ))}
+        </Row>
+        <Row className="mt-3">
+          <Col>
+            <div className="info-section">
+              <h3>Address</h3>
+              <Alert variant="dark">{address}</Alert>
+            </div>
+            <div className="info-section">
+              <h3>Public Key</h3>
+              <Alert variant="dark">{public_key}</Alert>
+            </div>
+          </Col>
+          <Col>
+            <div id="aat-info" className="mb-2">
+              <h3>AAT</h3>
+              <span>
+                <HelpLink size="2x" />
+                <p>How to create an AAT?</p>
+              </span>
+            </div>
+            <div className="aat-code">
+              <pre>
+                <code className="language-html" data-lang="html">
+                  {"# Returns\n"}
+                  <span id="aat">{jsonString}</span>
+                </code>
+                <p
+                  onClick={() => {
+                    // This won't be rendered on html
+                    const aat = document.querySelector("#aat").innerHTML;
+                    const el = document.createElement("textarea");
+
+                    el.value = aat;
+                    document.body.appendChild(el);
+                    el.select();
+                    document.execCommand("copy");
+                    document.body.removeChild(el);
```

Comment:
Please create an static function on Application Service at frontend for it, the function takes the value on item to copy.